lib/app/views/test_case.erb

```diff
@@ -70,10 +70,15 @@
                   <% if sequence_results[test_case.id].nil? %>
                     <%= test_case.sequence.test_count %> tests
                   <% else %>
-                    <%= sequence_results[test_case.id].required_passed %>/<%= sequence_results[test_case.id].required_total%> Required Tests Passed
+                    <%= sequence_results[test_case.id].required_passed %>/<%= sequence_results[test_case.id].required_total  - sequence_results[test_case.id].required_omitted %> Required Tests Passed
+
                     <% if sequence_results[test_case.id].optional_total > 0%> -
                       <%= sequence_results[test_case.id].optional_passed%>/<%= sequence_results[test_case.id].optional_total%> Optional Tests Passed
                     <% end%>
+
+                    <% if sequence_results[test_case.id].total_omitted > 0%> -
+                      <%= sequence_results[test_case.id].total_omitted %> Test <% if sequence_results[test_case.id].total_omitted != 1%>s<% end%> Omitted
```

Comment:
If we are going to make Test singular or plural based on the number of tests admitted then we should be consistent and do it for required and omitted tests too.

Reply:
![image](https://user-images.githubusercontent.com/51918208/61968169-4e526c80-afa5-11e9-8b7d-0f14bf0879be.png)

Reply:
I was thinking of if `0/1 Optional Tests Passed` should be singular, but looking at it the way it is now is fine because of the fraction I think it's fine the way it is.

Reply:
For future reference, since we use `ActiveSupport`, we have access to the super handy `String#pluralize` method:
```
[1] pry(#<TestInstanceTest>)> 'test'.pluralize
=> "tests"
[2] pry(#<TestInstanceTest>)> 'test'.pluralize(0)
=> "tests"
[3] pry(#<TestInstanceTest>)> 'test'.pluralize(1)
=> "test"
[4] pry(#<TestInstanceTest>)> 'test'.pluralize(2)
=> "tests"
```